silkworm/db/datastore/snapshots/snapshot_repository.hpp

```diff
@@ -107,7 +107,7 @@ class SnapshotRepository {
     }
 
     std::pair<std::optional<SegmentAndIndex>, std::shared_ptr<SnapshotBundle>> find_segment(
-        std::pair<datastore::EntityName, datastore::EntityName> names,
+        std::array<datastore::EntityName, 3> names,
```

Comment:
It would be better to insert a `using` declaration for `std::array<datastore::EntityName, 3>`, as it is used in 4 different places